GETTING_STARTED_KR.md

```diff
@@ -0,0 +1,128 @@
+# 시작하기
+
+euphony는 사용하기 굉장히 쉽습니다.
+
+이것은 두 부분으로 구성되어 있습니다.
+
+1) 송신기
+ - 음파를 생성하고 마이크를 통해 전송합니다.
+2) 수신기
+- 음파를 듣고 데이터를 수신합니다.
+
+## 송신기
+
+---
+
+### 음파를 생성하고 마이크를 통해 전송합니다.
+
+1. **데이터를 음파로 바꾸는 EuTxManager 객체를 생성**
+- Java
+
+```java
+EuTxManager mTxManager = new EuTxManager();
+```
+
+- Kotlin
+
+```kotlin
+val mTxManager = EuTxManager()
+```
+
+2. **euInitTransmit 메소드로 전송할 문장을 입력**
+- Java
+
+```java
+mTxManager.euInitTransmit("Hello, Euphony");
+```
+
+- Kotlin
+
+```kotlin
+mTxManager.euInitTransmit("Hello, Euphony")
+```
+
+3. **process 메소드로 문장(음파)을 몇 번 반복할지 설정 (-1은 무한반복)**
+- Java
+
+```java
+mTxManager.process(-1);
+```
+
+- Kotlin
+
+```kotlin
+mTxManager.process(-1)
+```
+
+## 수신기
+
+---
+
+### 레코더를 통한 음파 수신 부분
+
+> **RECORD_AUDIO 권한 필요**
+
+- AndroidManifest.xml에 작성한다.
+
+```xml
+<uses-permission android:name="android.permission.RECORD_AUDIO" />
+```
+
+1. **음파를 데이터로 받아오는 EuRxManager 객체를 생성**
+- Java
+
+```java
+EuRxManager mRxManager = new EuRxManager();
+```
+
+- Kotlin
+
+```kotlin
+val mRxManager = EuRxManager()
+```
+
+2. **음파를 받아오고 처리하는 Listener를 설정**
+- Java
+
+```java
+mRxManager.setAcousticSensor(new AcousticSensor() {
+	@Override
+	public void notify(String letters) {
+		// when data is received
+	}
+});
+```
+
+- Kotlin
+
+```kotlin
+mRxManager.acousticSensor = AcousticSensor { letters ->
+    //when data is received
```

Comment:
I think the indentation interval should be the same as the 90th line, too.
On the 90th line, it seems to use the tab button.